playground/nuxt.config.js

```diff
@@ -1,7 +1,8 @@
 export default defineNuxtConfig({
   modules: [
     [
-      "@storyblok/nuxt",
```

Comment:
Is this necessary? I would prefer to keep it as it was as it's using npm workspaces symlinks

Reply:
We need to make a stub build for development process (`nuxt-module-build stub`) if you like to use main package in playground.

Reply:
Sounds good. Let's do it this way then